apps/snitch_core/test/data/model/tax_rate_test.exs

```diff
@@ -38,12 +42,10 @@ defmodule Snitch.Data.Model.TaxRateTest do
   describe "update/2" do
     setup :tax_rate
 
-    test "successfuly calculator field", context do
-      params = %{calculator: "FlatRateCalculator"}
+    test "failed invalid calculator", context do
+      params = %{calculator: BinaryCalculator}
       %{tax_rate: tr} = context
-      assert {:ok, tr_new} = TaxRateModel.update(params, tr)
-      assert tr_new.id == tr.id
-      refute tr_new.calculator == tr.calculator
+      assert {:error, cs} = TaxRateModel.update(params, tr)
```

Comment:
1. Could you assert on the error message?
2. Is this `true` `tr.calculator == Snitch.Calculator.DefaultCalculator`? Or is this `tr.calculator == Elixir.Snitch.Calculator.DefaultCalculator`?

Reply:
`tr.calculator == Elixir.Snitch.Calculator.DefaultCalculator` because the module names are stored as such.